<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MeitalDaniel-PC" name="TestCases.Products_Test" tests="8" failures="1" timestamp="2020-06-15T17:47:45 IDT" time="13.463" errors="0">
  <testcase name="T03_addItemsToCart_1" time="2.475" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T03_addItemsToCart_4" time="2.411" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T03_addItemsToCart_5" time="2.338" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T03_addItemsToCart_2" time="2.391" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T02_getPageTitle" time="0.041" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T03_addItemsToCart_3" time="2.534" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T01_validLogin" time="0.793" classname="TestCases.Products_Test"/>
  <system-out/>
  <testcase name="T03_addItemsToCart_6" time="0.480" classname="TestCases.Products_Test">
    <failure type="java.lang.AssertionError" message="expected [6] but found [7]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [7]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at TestCases.Products_Test.T03_addItemsToCart_6(Products_Test.java:152)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
    </failure>
  </testcase> <!-- T03_addItemsToCart_6 -->
  <system-out/>
</testsuite> <!-- TestCases.Products_Test -->
